### PR TITLE
Add blogs to month of June 2022

### PR DESCRIPTION
**Pull request by Avarjana**
Add blogger name Avarjana Panditha to the list.
Add two blogs for the month of June 2022.

### CODE REVIEW

File: 2022.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## June (1)
```
Comment (somindatommy) [inline 2022.md:3]: Update the count here

File: 2022.md [hunk 2]
```diff
@@ -76,7 +78,6 @@
 * [Is gRPC going to take over REST?](https://medium.com/@thumilan/is-grpc-going-to-take-over-rest-d03896a9e1d3) by [Thumilan](https://medium.com/@thumilan)
 * [Securing your web applications with SAML 2.0 using WSO2 Identity Server](https://is-rajapaksha.medium.com/securing-your-web-applications-with-saml-2-0-using-wso2-identity-server-bb3db871d884) by [Isanka Rajapaksha](https://is-rajapaksha.medium.com/)
 * [ELK (Elastic) Stack Single Sign-On (SSO) with WSO2 Identity Server via OIDC](https://avarjana.medium.com/elk-elastic-stack-single-sign-on-sso-with-wso2-identity-server-via-oidc-43ce301ed27c) by [Avarjana Panditha](https://avarjana.medium.com/)
-* [Creating Complex “Transforms” in ELK (Elastic) Stack for WSO2 Identity Server Events](https://avarjana.medium.com/creating-complex-transforms-in-elk-elastic-stack-for-wso2-identity-server-events-176b430c0abd) by [Avarjana Panditha](https://avarjana.medium.com/)
```
Comment (somindatommy) [inline 2022.md:79]: Why have we removed this?

Comment (Avarjana) [reply 2022.md:79]: Duplicate line 58

File: 2022.md (hunk 1 above)
Comment (Avarjana) [reply 2022.md]: Resolved [9e0a1e7](https://github.com/wso2/iam-blogs/pull/130/commits/9e0a1e751f10f8f72d346cced000be7011204f38)

**Review by somindatommy — APPROVED** [review]

**Pull request merged**